bokeh/io/notebook.py

```diff
@@ -292,7 +292,7 @@ def push_notebook(*, document: Document | None = None, state: State | None = Non
             push_notebook(handle=handle)
 
     '''
-    from ..protocol import Protocol
+    from ..protocol import Protocol as BokehProtocol
```

Comment:
We might want to consider renaming `Protocol` for 3.0. The python `Protocol` type did not exist when this code was first written, but it makes things a big awkward now.

Reply:
On the other hand Python's `Protocol` could have been named `Interface` instead (`Protocol` seems to me a bit pompous for the job it does). Well, anyway, let's discuss this later on.